README.md

```diff
@@ -1,2 +1,3 @@
 # S47_Weirdy_Foody
-This is your first repository
+Weirdly Foody
+I got this idea from social medial specially food vloggers they used to take out different content from the weird food items so I thaught to make a site where they can see weird food items name and their pics and some Youtube videos where youu can see the reaction after eating that food and honest reviews of different peoples. There will be a specific section in which ingredients of that food will be present.
```

Comment:
The project description in the README file could be improved for readability by breaking it down into bullet points or separate sections. This will make it easier for others to understand the project's purpose and functionality. [medium]